torch/csrc/jit/export.cpp

```diff
@@ -51,6 +51,8 @@ void encodeTensor(onnx::TensorProto * p, const at::Tensor & tensor,
     p->add_dims(d);
   }
   onnx::DataType onnx_type;
+  // Most integral types and float16 need to be serialized as int32
```

Comment:
Why is float16 serialized as an integral number? 😮 Shouldn't it be float32 or just an error? It's unlikely that this is what anyone wants